client/src/components/SentimentAnalysis/SentimentAnalysis.css

```diff
@@ -80,3 +80,52 @@
 .bottom-padding {
   padding-bottom: 40px;
 }
+
+.similar-container {
+  width: 90%;
+  margin-right: 5%;
+  margin-left: 5%;
+  box-shadow: 0 8px 16px 0 rgba(0, 0, 0, 0.2);
+  padding-bottom: 40px;
+  border-radius: 10px;
+  margin-bottom: 40px;
+  justify-content: center;
+}
+
+.similar-card {
+  padding: 20px;
+}
+
+.similar-ad {
+  background-color: var(--primary-color);
+  border-radius: 5px;
+  padding: 10px;
+  padding-left: 20px;
+  padding-right: 20px;
+  margin: 20px;
+  width: 25%;
+}
+
+.similar-ad p {
+  color: var(--background-color);
+}
+
+.similar-ad a {
+  text-decoration: none;
+  color: var(--secondary-color);
+  transition: 0.5s;
+}
+
+.similar-ad a:hover {
+  color: black;
+  transition: 0.5s;
+}
+
+h4, #card-description {
+  text-align: center;
```

Comment:
Andrew will fight hard for this, just saying lol

Reply:
Haha that's ok I'm not super attached to it. If there's a better alternative I'll gladly use that instead

Reply:
The alternative is \<div style="display: flex; justify-content: center;"\>\<span>My text\>\</div\>.

For a header like this, text-align is fine.